e2e/test-page.html

```diff
@@ -126,6 +126,10 @@ <h3>SVG</h3>
   <img src='./assets/green.svg' class='image' />
   <h3>srcset</h3>
   <img srcset='./assets/yellow.gif, ./assets/red.png 2x' class='image' />
+  <h3>base64 SVG data: URL</h3>>
```

Comment:
`>>` 😱😉

Reply:
omg I'm the worst - have pushed a new version!